docs/nodes/validator-node/operator/node-requirements.md

```diff
@@ -18,23 +18,25 @@ To make your validator node and validator fullnode deployment hassle-free, make
 ## Hardware requirements
 
 For running an Aptos **validator node and validator fullnode** we recommend the following hardware resources:
+For running a validator and validator fullnode is that your hardware spec should be good enough to achieve a 30k peak
+TPS on mainnet. In order to evaluate that, you can use the command (https://gist.github.com/igor-aptos/298c87c0a5f095bdae4716b526d82520)
+It has two benchmarks, a shorter one (~10m) and a large db one (few hours). Once the script finishes, it will print out a table like: https://gist.github.com/igor-aptos/97d43db4f460ab4b29e4902cb3d31954, and column "t/s" is your achieved "transactions per second". Evaluation criteria is encoded in the script, and will warn you after the table if you've not met any of the criteria.
 
 - **CPU**:
-  - 8 cores, 16 threads
+  - 60 cores, 60 threads
   - 2.8GHz, or faster
-  - Intel Xeon Skylake or newer
-- **Memory**: 32GB RAM.
-- **Storage**: 2T SSD with at least 40K IOPS and 200MiB/s bandwidth.
+  - AMD Milan EPYC or Intel(R) Xeon(R) Platinum 
+- **Memory**: 64GB RAM.
+- **Storage**: 2T SSD with at least 60K IOPS and 200MiB/s bandwidth.
 - **Networking bandwidth**: 1Gbps
 
 ### Example machine types on various clouds
 
 - **AWS**
-  - c6id.8xlarge (if using a local SSD)
-  - c6i.8xlarge + io1/io2 EBS volume with 40K IOPS.
+  - c6id.16xlarge (if using a local SSD) 
```

Comment:
I think this is 32c64t?

Reply:
@grao1991 - `c6id.16xlarge` passed our benchmark for previewnet from the result - https://docs.google.com/spreadsheets/d/11U4k1JyTFIVOIhem9BfP-BReq1dAdCFcdN4TfpeS9mA/edit#gid=0